### ISSUE TITLE
[NTR] B6.SJL-Slc6a3tm1.1(cre)Bkmn/J knock-in mice

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#1266]
cc @rachadele
As per https://github.com/HumanCellAtlas/ontology/issues/92:
"UCSC would like to request a new HCA mouse ontology for the following strain:
https://www.jax.org/strain/006660
"DATIREScre knock-in mice have Cre recombinase expression directed to dopaminergic neurons, without disrupting endogenous dopamine transporter expression. These mice may be useful for studying gene function in dopaminergic neurons, such as drug addiction, Parkinson's disease and Attention Deficit-Hyperactivity Disorder (ADHD)."
We're wondering what term label/parent term would be most appropriate? Thanks!"

@rachadele, could you please provide your ORCID and, if possible, the reference of the study/article for which you need the term. Thanks. I'll look into the most appropriate parentage and will comment here when done - will aim at having the term ready in time for the next EFO release, scheduled for Oct 18th.

**Comment by paolaroncaglia** [comment]
Note for self: the Jax entry says "Strain of Origin | Not Specified", so I'll add as a subclass of 'Mus musculus'.

**Comment by paolaroncaglia** [comment]
- [ ] Add subset tag "added_for_HCA" to EFO_0030032 (B6.SJL-Slc6a3tm1.1(cre)Bkmn/J)
